psalm.xml

```diff
@@ -13,4 +13,12 @@
             <directory name="vendor" />
         </ignoreFiles>
     </projectFiles>
+    <issueHandlers>
+        <!-- Issue about &error; should go away with Psalm v6 -->
+        <UndefinedVariable>
+            <errorLevel type="suppress">
+                <file name="src/Doctrine/Instantiator/Instantiator.php" />
+            </errorLevel>
+        </UndefinedVariable>
+    </issueHandlers>
```

Comment:
The alternative here is to add `$error = null;` before this line: https://github.com/doctrine/instantiator/blob/f49f6a836a816609c853718ba2ea422dc18a8e4a/src/Doctrine/Instantiator/Instantiator.php#L169, but although it is surprising to have this undeclared variable being used here (at least to me), apparently, it works just fine.

Reply:
I wanted to test that and clone the repository, but it seems you already did that a minute ago :smile: